xml/ftp_banners.xml

```diff
@@ -668,6 +669,32 @@ more text</example>
     <param pos="0" name="os.product" value="VxWorks"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:windriver:vxworks:-"/>
   </fingerprint>
+  <fingerprint pattern="^[\w\-\.]* FTP server \((?:VxWorks\s?)+([\d\.]+)\) ready.$" flags="REG_ICASE">
+    <description>VxWorks 6 with version information</description>
+    <example os.version="6.6">NanoDAC FTP server (VxWorks VxWorks 6.6) ready.</example>
+    <example os.version="6.4">BVS-MR-BSC2 FTP server (VxWorks 6.4) ready.</example>
+    <param pos="0" name="os.vendor" value="Wind River"/>
+    <param pos="0" name="os.product" value="VxWorks"/>
+    <param pos="1" name="os.version"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:windriver:vxworks:-{os.version}"/>
+  </fingerprint>
+  <fingerprint pattern="^[\w&lt;&gt;]+\s*Tenor\sMultipath\sSwitch\sFTP\sserver\s\(Version VxWorks([\d\.]+)\) ready\.$" flags="REG_ICASE">
+    <description>VxWorks on Tenor MultiPath with version information</description>
```

Comment:
Is this hardware specific, can we add hw.product, etc?

Reply:
This is from sonar data, I wouldn't be confident of getting the correct hw.product etc set. For this I just wanted to make sure we got the vxworks version, and fairly confident from the banner alone that this will be correct.